### PR TITLE
Fix docs & scripts

### PR DESCRIPTION
**Pull request by mariacarmina**
## Description

Fix docs. Update scripts env vars. Removed deprecated exceptions.

## Is this PR related with an open issue?

Related to Issue #1005 

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: aquarius/events/events_monitor.py [hunk 1]
```diff
@@ -454,7 +454,7 @@ def get_last_processed_block(self):
                 try:
                     if self._es_instance.es.ping() is True:
                         break
-                except elasticsearch.exceptions.ElasticsearchException as es_err:
```
Comment (calina-c) [inline aquarius/events/events_monitor.py:457]: I think these should be treated separately. If it is not an ElasticsearchException, please log another error.

**Review by calina-c — CHANGES_REQUESTED** [review]
These all seem to be local changes, lgtm but are we sure the problem with indexing is not related to some server configuration? That should be in the scope of fixing the indexing. Did you compare your settings with @md00ux's live ones?

**Comment by mariacarmina** [comment]
> These all seem to be local changes, lgtm but are we sure the problem with indexing is not related to some server configuration? That should be in the scope of fixing the indexing. Did you compare your settings with @md00ux's live ones?

The settings from @md00ux didn't seem to need the env vars that I added in `exportlocal.sh`. I can revert changes from `exportlocal.sh` if so.

File: aquarius/events/events_monitor.py (hunk 1 above)
Comment (calina-c) [reply aquarius/events/events_monitor.py:457]: oh, ok then

**Review by calina-c — APPROVED** [review]

**Pull request merged**